tmt/cli.py

```diff
@@ -822,44 +824,11 @@ def init(context, path, template, force, **kwargs):
     * 'full' template contains a 'full' story, an 'full' plan and a shell test.
     """
 
-    # Check for existing tree
-    path = os.path.realpath(path)
-    try:
-        tree = tmt.Tree(path)
-        # Are we creating a new tree under the existing one?
-        if path == tree.root:
-            echo("Tree '{}' already exists.".format(tree.root))
-        else:
-            tree = None
-    except tmt.utils.GeneralError:
-        tree = None
-    # Create a new tree
-    if tree is None:
-        try:
-            fmf.Tree.init(path)
-            tree = tmt.Tree(path)
-        except fmf.utils.GeneralError as error:
-            raise tmt.utils.GeneralError(
-                "Failed to initialize tree in '{}': {}".format(
-                    path, error))
-        echo("Tree '{}' initialized.".format(tree.root))
-
-    # Populate the tree with example objects if requested
-    if template == 'empty':
-        non_empty_choices = [c for c in _init_template_choices if c != 'empty']
-        echo("To populate it with example content, use --template with "
-             "{}.".format(listed(non_empty_choices, join='or')))
-    else:
-        echo("Applying template '{}'.".format(template, _init_templates))
-    if template == 'mini':
-        tmt.Plan.create('/plans/example', 'mini', tree, force)
-    elif template == 'base':
-        tmt.Test.create('/tests/example', 'beakerlib', tree, force)
-        tmt.Plan.create('/plans/example', 'base', tree, force)
-    elif template == 'full':
-        tmt.Test.create('/tests/example', 'shell', tree, force)
-        tmt.Plan.create('/plans/example', 'full', tree, force)
-        tmt.Story.create('/stories/example', 'full', tree, force)
+    #import pdb
+    #pdb.set_trace()
+    tmt.base.Tree._save_context(context)
+    tmt.base.Tree.init(context, path, template, force, **kwargs)
```

Comment:
The `context` parameter is not used in the method and it's already stored into the `Tree` class on the line above, let's remove it.